nautobot_design_builder/design.py

```diff
@@ -842,7 +852,7 @@ def resolve_values(self, value: Union[list, dict, str]) -> Any:
                 value[k] = self.resolve_value(item)
         return value
 
-    def _create_objects(self, model_class: Type[ModelInstance], objects: Union[List[Any], Dict[str, Any]]):
+    def _create_or_import_objects(self, model_class: Type[ModelInstance], objects: Union[List[Any], Dict[str, Any]]):
```

Comment:
while this method is probably poorly named, I'm not sure that this PR should be where we rename it. Every change we have in the ltm branch is something we'll have to figure out how to pull over to the develop branch (which still doesn't have the lifecycle changes in it). I'd prefer to keep changes to absolute minimums in order to make the updates to `develop` easier down the road.

Reply:
That's a valid point. I will exchange it by a comment about it